### DISCUSSION TITLE
2011/06/13/%E8%B1%86%E8%B1%86%E7%9A%84%E7%88%AC%E8%A1%8C%E5%A7%BF%E5%8A%BF

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#332] [Announcements]
# 2011/06/13/%E8%B1%86%E8%B1%86%E7%9A%84%E7%88%AC%E8%A1%8C%E5%A7%BF%E5%8A%BF

周末突然发现豆豆可以肚皮离地，仅靠手和膝盖把身体支撑起来。不过豆豆的爷爷奶奶说，他两星期前就可以支起来了，只不过我白天都不在家，没有看到而已。

https://blog.qizhen.xyz/2011/06/13/%E8%B1%86%E8%B1%86%E7%9A%84%E7%88%AC%E8%A1%8C%E5%A7%BF%E5%8A%BF

**Comment by ruanqizhen** [comment]
之前博客下的留言：

ruanqizhen
2011年06月23日 12:25 编辑
相机是索尼的入门相机a550。不过据说镜头比相机更重要，我特地为豆豆买了一个专拍人像的 50 1.4 镜头。

回复
Haiying
2011年06月22日 19:25 编辑
小孩都是先往后退，后往前爬
你这相机不错呀，是那款?

回复
